package.json

```diff
@@ -72,10 +70,10 @@
         "CHANGELOG.md"
     ],
     "scripts": {
-        "lint": "eslint 'src/**/*.ts?(x)' --fix",
-        "lint:ci": "eslint 'src/**/*.ts?(x)'",
-        "format": "pretty-quick --staged",
-        "prepare": "husky install",
+        "lint": "eslint '{src,tests}/**/*.ts' --fix",
+        "lint:ci": "eslint '{src,tests}/**/*.ts'",
+        "format": "prettier --write '{src,tests}/**/*.ts'",
```

Comment:
we should probably exclude the auto-generated code from prettier and the linter since we aren't manually maintaining it. ie everything in `src/generated`

Reply:
prettier already ignored it, but i made it more specific and added the ignore to eslint as well [6521962](https://github.com/passageidentity/passage-node/pull/168/commits/6521962fcf414c97f65aa7ecd1024229fd7419c3)

Reply:
Eslint is now complaining about the "ignores" field in its own config file which is odd

Reply:
Also I notice that the openAPI generator already put eslint-disable statements at the top of all their files so the linter is already ignoring them too

Reply:
whoops, sorry that field was only valid in v9.x. switched it to a v8.x field [2a08e2c](https://github.com/passageidentity/passage-node/pull/168/commits/2a08e2c61f8aabf70478800014a18c9c25dd9cb8)